index.html

```diff
@@ -636,7 +636,7 @@ <h2 style="color:yellow">
 	<div class="footer">
 		<div class="container">
 			<div class="copy-right">
-				<p> &copy; 2020 Ping Foundation</p>
+				<p> &copy; 2023 Ping Foundations</p>

```

Comment:
Insted of hard coding 2023, we can dynamically display year using javascript Date() object, so that we don't have to update it every year.What do you think @Vasudv27 ?
```suggestion
				
<p id='display-year'></p>
```
``` in javascript
				const year=document.getElementById('display-year') ;
                                       year.innerHTML=new Date().getFullYear() ;

```